.edm4hep-ci.d/compile_and_test_with_podio.sh

```diff
@@ -4,14 +4,14 @@ git clone --depth 1 https://github.com/AIDASoft/podio || true
 cd podio
 mkdir build install
 cd build
-cmake -DCMAKE_CXX_STANDARD=17 -DBUILD_TESTING=OFF -DCMAKE_INSTALL_PREFIX=../install -G Ninja .. \
+cmake -DCMAKE_CXX_STANDARD=17 -DBUILD_TESTING=OFF -DCMAKE_INSTALL_PREFIX=$PWD/../install -DENABLE_SIO=ON -G Ninja .. \
       && ninja -k0 \
       && ninja install \
           || exit 1
 
 cd ..
 export CMAKE_PREFIX_PATH=$PWD/install:$CMAKE_PREFIX_PATH
-export PODIO=$PWD/install:$CMAKE_PREFIX_PATH
+export PODIO=$PWD/install  # $CMAKE_PREFIX_PATH
```

Comment:
```suggestion

```

Actually this env var is not (should not?) be used for CMake.

Reply:
So could it be removed in this setup? Or is it needed by something else?

Reply:
Yes, this can be removed.